README.org

```diff
@@ -1,6 +1,9 @@
 #+author: Cam Parra
 
-* Sovrin Ledger Payment plugins
+#+ATTR_HTML: :style margin-left: auto; margin-right: auto;
+[[./banner.png]]
+
+* Sovrin Token Plugins
```

Comment:
did we decide on calling this "Sovrin Token Plugins"?

Reply:
yes